slider/mod.py

```diff
@@ -147,9 +147,29 @@ def circle_radius(cs):
     return (512 / 16) * (1 - 0.7 * (cs - 5) / 5)
 
 
+class HitWindows(namedtuple('HitWindows', 'perfect, good, bad')):
+    """Times to hit an object at various accuracies
+
+    Parameters
+    ----------
+    perfect : int
```

Comment:
I thought they had been, but I think that was other rhythm games.

Perhaps hit_300 or h300 etc.? I don't like the [300] because I don't think that looks like an attribute access (which this is).

Maybe we should just drop the named tuple entirely and just return a normal tuple? That would certainly solve the problem of names.